lessons/Early-Math/associative-property/langs/en-us/pages/discover-assoc.adoc

```diff
@@ -0,0 +1,66 @@
+= Discover the Associative Property
+
+++++
+<style>
+  table {grid-template-rows: 3fr 1fr !important;}
+  div.circleevalsexp .value,
+  div.circleevalsexp .studentBlockAnswerFilled { min-width:unset; }
+</style>
+++++
+
+@n Take a look at the two Circles of Evaluation below. Now, evaluate them to help you decide if the Associative Property holds for problems involving *addition*.
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="^.^3,^.^3"]
+|===
+|@show{(coe '(+ (+ 10 8) 3))} | @show{(coe  '(+ 10 (+ 8 3)))}
+| @ifnotsoln{@math{(10 + 8) +3 = ?}} @ifsoln{@math{(10 + 8) + 3 = 21}} | @ifnotsoln{@math{10 + (8 + 3) = ?}} @ifsoln{@math{10 + (8 + 3) = 21}}
+|===
+
+What do you notice? @fitb{}{@ifsoln{Both Circles evaluate to 21. The first Circle has the sum of 10 and 8 in a nested Circle, while the second}}
+
+@fitb{}{@ifsoln{Circle has the sum of 8 and 3 in a nested Circle.}}
+
+@vspace{1ex}
+
+These Circles represent the Associative Property of Addition! *The Associative Property is true for any expression where all possible _groupings_ produce the same result.* Draw another example of the Associative Property of Addition with any *four* numbers, below. Evaluate each Circle to confirm that they are equivalent.
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="^.^3,^.^3"]
+|===
```

Comment:
I think it's unclear what belongs in this table. Want to add some words? Perhaps the first column of each table on this page should be titles for each row? eg 1) Circle of Evaluation 2) Evaluate the Expression

Reply:
I experimented with this and it looked super awkward. I think it has to be clear enough, based on the example and the directions.